docs/bundles/SyliusGridBundle/configuration.rst

```diff
@@ -32,15 +30,38 @@ Here you will find all configuration options of ``sylius_grid``.
                     name:
                         type: string # Type of filter
                         label: app.ui.name
+                        enabled: true
+                        template: ~
+                        position: 100
                         options: { }
                 actions:
                     main:
                         create:
                             type: create
+                            label: sylius.ui.create
+                            enabled: true
+                            icon: ~
+                            position: 100
```

Comment:
Do we need to specify `position` and `enabled: true`? IMO these should be default values.